### ISSUE TITLE
f.map("Boston, MA") shows nothing on Windows

### ISSUE THREAD

**Issue by jjgh** [zach-capalbo/flammarion#30]
Hi guys, 
this does not show anything on Windows 7 64bit

`f.map("Boston, MA")`

Am I supposed to *require* something else other than flammarion?

```
> ruby -v
> ruby 2.2.2p95 (2015-04-13 revision 50295) [x64-mingw32]
```

**Comment by zach-capalbo** [comment]
Do any of the other examples work? Are you running this from inside irb or from a script?

**Comment by jjgh** [comment]
Not inside of IRB,  running it as a script. 
"Message Composer with Address Book" example works although panes background, buttons background, text areas background and window background look black instead of white.
I'll post a scrrenshot later.

**Comment by mattscilipoti** [comment]
Same issue on OSX (10.11.6) .  Everything is working, in irb or as script (with `f.wait_until_closed`), except for `f.map`.

Using flammarion 0.1.11

**Comment by zach-capalbo** [comment]
Hmm... If you press Ctrl+shift+i on the window that pops up, are there any errors reported in the web console?

**Comment by mattscilipoti** [comment]
I don't see any change when I press Ctrl+shift+i.  It creates an email when I press I press cmd+shift+i.

**Comment by zach-capalbo** [comment]
Thanks for the report! I've reproduced the issue, and am working on a fix now.

**Comment by zach-capalbo** [comment]
Fixed in version 0.1.13
